src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1591,6 +1638,7 @@ void goto_instrument_parse_optionst::help()
     " --dump-cpp                   generate C++ source\n"
     " --dot                        generate CFG graph in DOT format\n"
     " --interpreter                do concrete execution\n"
+    " --create-function-stub fun   generate a modular verification stub for fun, using its contract\n" // NOLINT(*)
```

Comment:
This option should probably have a name that's more specific to contracts, function stub can mean lots of things.